__tests__/routes/article.test.js

```diff
@@ -79,12 +79,13 @@ describe('articles', () => {
   });
 
   test('should return article', async () => {
-    expect.assertions(5);
+    expect.assertions(6);
     const res = await request(app).get(`${urlPrefix}/articles/${newArticle.slug}`);
     expect(res.status).toBe(200);
     expect(res.body).toBeDefined();
```

Comment:
not necessary

Reply:
@Nziranziza It came from the previous work but I am going to remove it. Thank you for the feedback as we shall always refactor some part of the codebase when necessary